.github/workflows/refresh-samples.yaml

```
@@ -6,6 +6,8 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'scripts/**.ts'
```

It should also update when a mapper is modified.
```suggestion
      - 'scripts/**.ts'
      - 'mappers/**.ts'
```